CONTRIBUTORS.md

```diff
@@ -4,3 +4,4 @@ cytominer-database initially developed by
 - [Allen Goodman](https://github.com/0x00b1/)
 - [Claire McQuin](https://github.com/mcquin/)
 - [Shantanu Singh](https://github.com/shntnu/)
+- [Frances Hubis](https://github.com/diskontinuum/)
```

Comment:
```suggestion
- [Gregory Way](https://github.com/gwaygenomics)
- [Frances Hubis](https://github.com/diskontinuum/)
```

Reply:
I hadn't noticed this file, thanks for adding yourself - with this suggested change, I also add myself.